data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,847 @@ mission "Coalition License Patch"
+			`	With it, there's a message. It's from Inturi and Aunaris, the Saryd couple who you helped move out to their very own home.`
+			`	In the message, they apologize for taking so long to pay you, and go on to tell you about how they've been doing there. Aunaris has been working in a school there, while Inturi has been helping other residents set up their own "ship-houses," as he calls them. Apparently it's become somewhat popular there after people saw the couple's home.`
+			`	They thank you again and say you're welcome to visit anytime.`
+
+
+
+mission "Saryd Couple Follow-Up B"
+	invisible
+	landing
+	to offer
+		has "event: saryd couple follow-up"
+		has "refused saryd couple payment"
+	source
+		government "Coalition" "Heliarch"
+	on offer
+		conversation
+			`When you land, you see you've received a message. It's from Inturi and Aunaris, the Saryd couple who you helped move out to their very own home.`
+			`	Along with the message, they've sent a picture of them both tending to some flowers and short trees in a little garden they've started up near the ship. In the message they go on to tell you about how they've been doing there. Aunaris has been working in a school there, while Inturi has been helping other residents set up their own "ship-houses," as he calls them. Apparently it's become somewhat popular there after people saw the couple's home.`
+			`	They thank you again and say you're welcome to visit anytime.`
+
+
+
+mission "Longcow Antibiotics 1"
+	minor
+	name "Longcow Antibiotics"
+	description "Arachi ranchers have asked you to pick up <cargo> on <stopovers>, then deliver them to <destination>."
+	cargo "Longcow Antibiotics" 7
+	to offer
+		has "Coalition: First Contact: done"
+		random < 35
+	source
+		attributes arach
+	stopover "New Finding"
+	destination "Corral of Meblumem"
+	on offer
+		require "Jump Drive"
+		conversation
+			`You see some Arachi talking among themselves in the spaceport, occasionally glancing at and pointing at you.`
+			`	One of them finally comes to you and says, "Greetings, Captain. Interested in performing a cargo mission for us, are you?"`
+			`	She tells you that they are a cooperative of Arachi ranchers, explains that the safety and health regulations for their business, longcow ranching, were recently updated, and that they need someone to pick up the newest antibiotics for the cattle on <destination>.`
+			`	"A jump drive you have, so get them faster - and allow us to resume business faster - you would."`
+			choice
+				`	"You've found the perfect person for the job. I'm in."`
+				`	"I suppose that's true, but I'm not looking for courier jobs, sorry."`
+					decline
+			`	The other Arachi come over to thank you after she signals to them that you've accepted to help. They explain that after you've picked up the antibiotics on <planet>, you'll have to drop them off on another planet before wishing you a good trip.`
```

Comment:
So you've changed this mission to use a stopover, but this wording here is still very roundabout. 
```suggestion
			`	The other Arachi come over to thank you after she signals to them that you've accepted to help. They explain that after you've picked up the antibiotics on <stopovers>, you'll have to drop them off on <planet> before wishing you a good trip.`
```